sql/src/test/java/io/crate/integrationtests/UpdateIntegrationTest.java

```diff
@@ -1001,4 +1003,30 @@ public void test_update_by_query_returning_multiple_results() throws Exception {
         assertThat(response.rows()[1][1], is(3L));
         assertThat(response.rows()[1][2], is("updated"));
     }
+
+    @Test
+    public void test_update_sys_tables_returning_count() throws Exception {
+        execute("update sys.node_checks set acknowledged = true where id = 1");
```

Comment:
Shouldn't a test like this already exist somewhere?

Reply:
You are right, it is `SysNodeCheckerIntegrationTest`. Thanks for spotting this.